src/validate.ts

```diff
@@ -10,7 +11,7 @@ async function runTest(): Promise<void> {
   try {
     client = new MongoClient(connectionString, { useUnifiedTopology: true });
     await client.connect();
-    console.log(`Client connected: ${client.isConnected()}`);
+    console.log(`Client connected to "${connectionString}": ${client.isConnected()}`);
```

Comment:
Couldn't this potentially be an issue? Having the `connectionString` put into a `console.log`. I'm more thinking on the public repos as then it may expose something?